manifests/install/charts/as-a-second-scheduler/values.yaml

```diff
@@ -23,3 +23,33 @@ plugins:
   enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","NodeResourcesAllocatable"]
   disabled: ["PrioritySort"] # only in-tree plugins need to be defined here
 
+pluginConfig:
+  plugins:
+    - Coscheduling
+    - NodeResourceTopologyMatch
+    - NodeResourcesAllocatable
+    - LoadVariationRiskBalancing
+    - TargetLoadPacking
```

Comment:
Is it possible to remove this section? ideally, we should refrain from adding new "formats" of config if unnecessary.

In other words, does it technically work by incorporating the original pluginConfig:

```yaml
pluginConfig:
- name: foo
  args:
- name: bar
  args:
```

Reply:
That's a good point, I've done some changes, PTAL.